src/cli/output/installDependencies.ts

```diff
@@ -14,7 +16,24 @@ import installPackages from "../../utils/installPackages.js";
  */
 export default async function installDependencies(input: UserInput) {
   const { plugins, projectDir, appConfig } = input;
-
+   const pkgManager = getPackageManager();
```

Comment:
It seems that your prettier is not set up correctly in VS Code, it should be formatting according to .prettierrc in the project. 

You can also always run the formatting manually, but I have a format on save settings on.

Reply:
My code for you project is formatted with prettier Eslint. I try to think to make a prettier fix from the application before pushing the code. I'm not to used to it.